### PR TITLE
Adds species specific ass pics

### LINKED ISSUE #29: Missing Ass Photos
There are only human ass/drone/"alien" ass photos no 4rry butts

### PR DESCRIPTION
**Pull request by Birdtalon**
Fixes #29

Adds `icons/mob/ass.dmi` for ass pictures, migrates existing ass pictures to this file.
Adds `ass_pic` var to species datums set to the icon name of the ass in question. If there is no ass it defaults to the human ass.

Human
dreammaker_2017-09-27_18-33-08

Alien
dreammaker_2017-09-27_18-33-46

Drone
dreammaker_2017-09-27_18-33-49

Adds ass pictures for:

- Ethari _(needs updated sprite)_
dreammaker_2017-09-27_18-33-37
- Slime
dreammaker_2017-09-27_18-33-29
- Unathi
dreammaker_2017-09-27_18-33-22
- Plasmaman
dreammaker_2017-09-27_18-33-42

from Paradise.

@FlattestGuitar has asked for a new ass pic to be sprited for Ethari as a mixture between the two possible Ethari types. **This is still required** and this uses the Paradise vulpkanin ass for now.

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
To be fair, I think the Ethari ass sprite is fine. It fits all the tails out there. I'd let it be.

**Comment by Birdtalon** [comment]
Well if you're happy with it then so am I.

**Comment by EldritchSigma** [comment]
Picture no.4 is looking good. Very good in fact.

**Comment by Lady-Luck** [comment]
I don't like the unathi one, makes their butts look like they have ugly fat dimples all over them. :C

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm [hunk 1]
```diff
@@ -18,6 +18,7 @@
 	exotic_bloodtype = "L"
 	disliked_food = GRAIN | DAIRY
 	liked_food = GROSS | MEAT
+	ass_pic = "unathi"
```
Comment (Lady-Luck) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: Use standard male/female pics? There doesn't seem to be a gender distinction for unathi and the butt image looks very ugly.

**Comment by FlattestGuitar** [comment]
If someone makes us better sprites, we'll use them. Otherwise, 👌 🍑

**Pull request merged**